app/pb_kits/playbook/pb_icon_body_title/_icon_body_title.jsx

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+import Title from "../pb_title/_title.jsx";
+import Body from "../pb_body/_body.jsx";
+import Icon from "../pb_icon/_icon.jsx";
+
+type IconBodyTitleProps = {
+  body?: String,
+  className?: String,
+  data?: String,
+  icon?: String,
+  id?: String,
+  title?: String,
+  link?: String,
+}
+
+const IconBodyTitle = ({ body, className, data, icon, id, title, link }: IconBodyTitleProps) => {
+  const titleContent = !!link ? (<a href={link}>{title}</a>) : title 
+  return (
+    <div className={className || `pb_icon_body_title_kit_${icon}`} id={id} data={data}>
```

Comment:
Here, we want to leverage the `classnames` library to optionally add any additional `className`, but always output `pb_icon_body_title_kit_${icon}`. So the update here might look like:

```jsx
<div className={classnames(`pb_icon_body_title_kit_${icon}`, className)} ...
```